app/loggers/log_worker_queue_stats.rb

```diff
@@ -0,0 +1,30 @@
+class LogWorkerQueueStats
+  def self.perform
+    record_totals
+    record_queue_stats
+  end
+
+  private
+
+  def self.record_totals
```

Comment:
if you use the other class method syntax you can avoid using `private_class_method`:

```ruby
class LogWorkerQueueStats
  class << self
    def perform
      record_totals
      record_queue_stats
    end

    private

    def record_totals
      log_to_datadog("sidekiq.queues.total_size", Sidekiq::Queue.all.map(&:size).sum)
      log_to_datadog("sidekiq.queues.total_workers", Sidekiq::Workers.new.size)
    end

...
end
```